### PR TITLE
Removes SentryReportJob and endpoint that uses it

### PR DESCRIPTION
**Pull request by jennysivapalan**
We are getting a lot of errors in Teamcity for downloading to postgres jar. I looked into and it's used for `SentryReportJob` which is used on admin for the  `/analytics/content/gallery`. This endpoint is returning `Sentry reports not generated` and I don't believe it is being used.

### CODE REVIEW

File: project/Frontend.scala [hunk 1]
```diff
@@ -103,11 +103,9 @@ object Frontend extends Build with Prototypes {
 
   val admin = application("admin").dependsOn(commonWithTests).aggregate(common).settings(
     libraryDependencies ++= Seq(
-      postgres,
       paClient,
       dfpAxis,
       anormModule,
-      jdbc,
```
Comment (jennysivapalan) [inline project/Frontend.scala]: I might have been delete happy here, is `jdbc` used? It compiles and runs locally fine.

Comment (gklopper) [inline project/Frontend.scala]: Should be fine if it works locally

**Comment by gklopper** [comment]
:ship: :+1:

**Comment by johnduffell** [comment]
:+1: do it!! what's the worst that could happen? =) :joy: :speedboat: :zap:

**Comment by NathanielBennett** [comment]
:+1: I spent hours trying to figure out that SQL ..

**Comment by jennysivapalan** [comment]
Sorry @NathanielBennett!

**Pull request merged**